modules/standard/IO.chpl

```diff
@@ -1631,50 +1631,86 @@ proc file.fsync() throws {
   if err then try ioerror(err, "in file.fsync", this._tryGetPath());
 }
 
+/*
+  A compile-time parameter to control the behavior of :proc:`file.path`
+
+  When 'false', the deprecated behavior is used (i.e., return the relative path
+  whenever available)
```

Comment:
Technically I believe it's the "shortest" path between the relative and absolute paths.

Reply:
Good point, I'll fix that wording. Thanks!